content/doc/book/using/long-running-agent-process.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: section
+title: Long running agent process
+---
+
+= Long running agent process
+
+To reliably kill processes spawned by a job during a build, Jenkins
+contains a bit of native code to list up such processes and kill them.
+This is tested on several platforms and architectures, but if you find a show-stopper problem because of this, you can disable this feature by setting a Java property named "hudson.util.ProcessTree.disable" to the value "true".
```

Comment:
Link to the new documentation on system properties.

```suggestion
This is tested on several platforms and architectures, but if you find a show-stopper problem because of this, you can disable this feature by setting the Java property link:/doc/book/managing/system-properties/#hudson-util-processtree-disable["hudson.util.ProcessTree.disable"] to the value "true".
```